easybuild/easyconfigs/g/gnuplot/gnuplot-5.0.1-intel-2015b.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg/Luxembourg Centre for Systems Biomedicine
+# Authors::   Fotis Georgatos <fotis.georgatos@uni.lu>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-97.html
+##
+easyblock = 'ConfigureMake'
+
+name = 'gnuplot'
+version = '5.0.1'
+
+homepage = 'http://gnuplot.sourceforge.net/'
+description = """Portable interactive, function plotting utility"""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [('http://sourceforge.net/projects/gnuplot/files', 'download')]
+
+dependencies = [
+    ('cairo', '1.14.4'),
+    ('libjpeg-turbo', '1.4.2'),
+    ('libgd', '2.1.1'),
```

Comment:
@wpoely86:

```
-bash-4.1$ echo "help terminal" | gnuplot
Gnuplot supports a large number of output formats. These are selected by
choosing an appropriate terminal type, possibly with additional modifying
options. See `set terminal`.

This document may describe terminal types that are not available to you
because they were not configured or installed on your system. To see a list of
terminals available on a particular gnuplot installation, type 'set terminal'
with no modifiers.

Subtopics available for terminal:
   cairolatex        canvas            cgm               context
   corel             dumb              dxf               eepic
   emf               emtex             epscairo          epslatex
   fig               gif               hpgl              jpeg
   latex             mf                mp                pcl5
   pdfcairo          png               pngcairo          pop
   postscript        pslatex           pstex             pstricks
   push              qms               qt                svg
   tek40xx           tek410x           texdraw           tgif
   tkcanvas          tpic              vttek             x11
   xlib              xterm
```

both `png` and `pngcairo` are listed